### PR TITLE
merging spin into automaticRoute

### PR DESCRIPTION
**Pull request by bill-n**

### CODE REVIEW

File: src/app/navigate/navigate.component.ts [hunk 1]
```diff
@@ -25,10 +25,11 @@ export class NavigateComponent implements OnInit {
       this.authenticationCode = queryParam.get("code");
       this.oidc_redirect_path = this.openId.redirect_page_to_oidc;
     });
```
Comment (patriciakasapa) [inline src/app/navigate/navigate.component.ts]: I thought you were to bring these guys, }); down or ?

File: src/app/app.component.html [hunk 2]
```diff
@@ -12,4 +12,8 @@ <h5>Turntabl</h5>
     </div>
   </nav>
 </div>
+<div class="lds-ripple">
```
Comment (patriciakasapa) [inline src/app/app.component.html]: Eeerrrmmm... do these divs have any significant functionalities?

File: src/app/navigate/navigate.component.ts (hunk 1 above)
Comment (patriciakasapa) [inline src/app/navigate/navigate.component.ts]: Eeerrrmm... i thought you brought these guys });  down or?

File: src/app/app.component.html (hunk 2 above)
Comment (bill-n) [reply src/app/app.component.html]: naa you can delete it...it is a dead code

**Pull request merged**